2020-08-22 08:56:43.9391 ERROR System.Threading.ThreadAbortException: Subproceso anulado.
   en System.Threading.Thread.AbortInternal()
   en System.Threading.Thread.Abort(Object stateInfo)
   en System.Web.HttpResponse.AbortCurrentThread()
   en System.Web.HttpResponse.End()
   en System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)
   en System.Web.HttpResponse.Redirect(String url)
   en SBS.UIF.CONTRALAFT.Web.pages.Login.Submit_Login(Object sender, EventArgs e) en D:\proynet\fsd2\SBS.UIF.CONTRALAFT.Web\pages\login.aspx.cs:línea 32 Subproceso anulado.
2020-08-22 09:04:51.1381 ERROR IBatisNet.Common.Exceptions.ConfigurationException: 
- The error occurred while loading SqlMap.
- initialize type alias
- The error occurred in <sqlMap resource="Recursos/core/Usuario.xml" xmlns="http://ibatis.apache.org/dataMapper" />.  
- Check the PE.GOB.FSD.Entity.Core.Usuario,PE.GOB.FSD.Entity. ---> System.TypeLoadException: Could not load type from string value 'PE.GOB.FSD.Entity.Core.Usuario,PE.GOB.FSD.Entity'. ---> System.IO.FileNotFoundException: No se puede cargar el archivo o ensamblado 'PE.GOB.FSD.Entity' ni una de sus dependencias. El sistema no puede encontrar el archivo especificado.
   en System.Reflection.RuntimeAssembly._nLoad(AssemblyName fileName, String codeBase, Evidence assemblySecurity, RuntimeAssembly locationHint, StackCrawlMark& stackMark, IntPtr pPrivHostBinder, Boolean throwOnFileNotFound, Boolean forIntrospection, Boolean suppressSecurityChecks)
   en System.Reflection.RuntimeAssembly.nLoad(AssemblyName fileName, String codeBase, Evidence assemblySecurity, RuntimeAssembly locationHint, StackCrawlMark& stackMark, IntPtr pPrivHostBinder, Boolean throwOnFileNotFound, Boolean forIntrospection, Boolean suppressSecurityChecks)
   en System.Reflection.RuntimeAssembly.InternalLoadAssemblyName(AssemblyName assemblyRef, Evidence assemblySecurity, RuntimeAssembly reqAssembly, StackCrawlMark& stackMark, IntPtr pPrivHostBinder, Boolean throwOnFileNotFound, Boolean forIntrospection, Boolean suppressSecurityChecks)
   en System.Reflection.RuntimeAssembly.InternalLoad(String assemblyString, Evidence assemblySecurity, StackCrawlMark& stackMark, IntPtr pPrivHostBinder, Boolean forIntrospection)
   en System.Reflection.RuntimeAssembly.InternalLoad(String assemblyString, Evidence assemblySecurity, StackCrawlMark& stackMark, Boolean forIntrospection)
   en System.Reflection.Assembly.Load(String assemblyString)
   en IBatisNet.Common.Utilities.TypesResolver.TypeResolver.LoadTypeDirectlyFromAssembly(TypeAssemblyInfo typeInfo)
   en IBatisNet.Common.Utilities.TypesResolver.TypeResolver.ResolveType(String typeName)
   --- Fin del seguimiento de la pila de la excepción interna ---
   en IBatisNet.Common.Utilities.TypesResolver.TypeResolver.ResolveType(String typeName)
   en IBatisNet.Common.Utilities.TypesResolver.TypeResolver.Resolve(String typeName)
   en IBatisNet.Common.Utilities.TypesResolver.CachedTypeResolver.Resolve(String typeName)
   en IBatisNet.Common.Utilities.TypeUtils.ResolveType(String typeName)
   en IBatisNet.DataMapper.Configuration.Serializers.TypeAliasDeSerializer.Deserialize(XmlNode node, ConfigurationScope configScope)
   en IBatisNet.DataMapper.Configuration.DomSqlMapBuilder.ConfigureSqlMap()
   en IBatisNet.DataMapper.Configuration.DomSqlMapBuilder.Initialize()
   en IBatisNet.DataMapper.Configuration.DomSqlMapBuilder.Build(XmlDocument document, DataSource dataSource, Boolean useConfigFileWatcher, Boolean isCallFromDao)
   --- Fin del seguimiento de la pila de la excepción interna ---
   en IBatisNet.DataMapper.Configuration.DomSqlMapBuilder.Build(XmlDocument document, DataSource dataSource, Boolean useConfigFileWatcher, Boolean isCallFromDao)
   en IBatisNet.DataMapper.Configuration.DomSqlMapBuilder.ConfigureAndWatch(String resource, ConfigureHandler configureDelegate)
   en PE.GOB.FSD.DataAccess.Mapper.MapperPro.InitMapper() en D:\proynet\fsd2\SBS.UIF.CONTRALAFT.DataAccess\Mapper\MapperPro.cs:línea 20
   en PE.GOB.FSD.DataAccess.Mapper.MapperPro.Instance() en D:\proynet\fsd2\SBS.UIF.CONTRALAFT.DataAccess\Mapper\MapperPro.cs:línea 31
   en PE.GOB.FSD.DataAccess.BaseService`1.QueryForObject(String nombreMetodo, Object map, String mapper) en D:\proynet\fsd2\SBS.UIF.CONTRALAFT.DataAccess\BaseDatos.cs:línea 20
   en PE.GOB.FSD.DataAccess.Core.UsuarioDataAccess.buscarUsuario(Usuario _usuario) en D:\proynet\fsd2\SBS.UIF.CONTRALAFT.DataAccess\Core\UsuarioDataAccess.cs:línea 19
   en PE.GOB.FSD.BusinessLogic.Core.UsuarioBusinessLogic.buscarUsuario(Usuario _usuario) en D:\proynet\fsd2\SBS.UIF.CONTRALAFT.BusinessLogic\Core\UsuarioBusinessLogic.cs:línea 27
   en PE.GOB.FSD.Web.pages.Login.Submit_Login(Object sender, EventArgs e) en D:\proynet\fsd2\SBS.UIF.CONTRALAFT.Web\pages\login.aspx.cs:línea 30 
- The error occurred while loading SqlMap.
- initialize type alias
- The error occurred in <sqlMap resource="Recursos/core/Usuario.xml" xmlns="http://ibatis.apache.org/dataMapper" />.  
- Check the PE.GOB.FSD.Entity.Core.Usuario,PE.GOB.FSD.Entity.
2020-08-22 09:04:56.3999 ERROR IBatisNet.Common.Exceptions.ConfigurationException: 
- The error occurred while loading SqlMap.
- initialize type alias
- The error occurred in <sqlMap resource="Recursos/core/Usuario.xml" xmlns="http://ibatis.apache.org/dataMapper" />.  
- Check the PE.GOB.FSD.Entity.Core.Usuario,PE.GOB.FSD.Entity. ---> System.TypeLoadException: Could not load type from string value 'PE.GOB.FSD.Entity.Core.Usuario,PE.GOB.FSD.Entity'. ---> System.IO.FileNotFoundException: No se puede cargar el archivo o ensamblado 'PE.GOB.FSD.Entity' ni una de sus dependencias. El sistema no puede encontrar el archivo especificado.
   en System.Reflection.RuntimeAssembly._nLoad(AssemblyName fileName, String codeBase, Evidence assemblySecurity, RuntimeAssembly locationHint, StackCrawlMark& stackMark, IntPtr pPrivHostBinder, Boolean throwOnFileNotFound, Boolean forIntrospection, Boolean suppressSecurityChecks)
   en System.Reflection.RuntimeAssembly.nLoad(AssemblyName fileName, String codeBase, Evidence assemblySecurity, RuntimeAssembly locationHint, StackCrawlMark& stackMark, IntPtr pPrivHostBinder, Boolean throwOnFileNotFound, Boolean forIntrospection, Boolean suppressSecurityChecks)
   en System.Reflection.RuntimeAssembly.InternalLoadAssemblyName(AssemblyName assemblyRef, Evidence assemblySecurity, RuntimeAssembly reqAssembly, StackCrawlMark& stackMark, IntPtr pPrivHostBinder, Boolean throwOnFileNotFound, Boolean forIntrospection, Boolean suppressSecurityChecks)
   en System.Reflection.RuntimeAssembly.InternalLoad(String assemblyString, Evidence assemblySecurity, StackCrawlMark& stackMark, IntPtr pPrivHostBinder, Boolean forIntrospection)
   en System.Reflection.RuntimeAssembly.InternalLoad(String assemblyString, Evidence assemblySecurity, StackCrawlMark& stackMark, Boolean forIntrospection)
   en System.Reflection.Assembly.Load(String assemblyString)
   en IBatisNet.Common.Utilities.TypesResolver.TypeResolver.LoadTypeDirectlyFromAssembly(TypeAssemblyInfo typeInfo)
   en IBatisNet.Common.Utilities.TypesResolver.TypeResolver.ResolveType(String typeName)
   --- Fin del seguimiento de la pila de la excepción interna ---
   en IBatisNet.Common.Utilities.TypesResolver.TypeResolver.ResolveType(String typeName)
   en IBatisNet.Common.Utilities.TypesResolver.TypeResolver.Resolve(String typeName)
   en IBatisNet.Common.Utilities.TypesResolver.CachedTypeResolver.Resolve(String typeName)
   en IBatisNet.Common.Utilities.TypeUtils.ResolveType(String typeName)
   en IBatisNet.DataMapper.Configuration.Serializers.TypeAliasDeSerializer.Deserialize(XmlNode node, ConfigurationScope configScope)
   en IBatisNet.DataMapper.Configuration.DomSqlMapBuilder.ConfigureSqlMap()
   en IBatisNet.DataMapper.Configuration.DomSqlMapBuilder.Initialize()
   en IBatisNet.DataMapper.Configuration.DomSqlMapBuilder.Build(XmlDocument document, DataSource dataSource, Boolean useConfigFileWatcher, Boolean isCallFromDao)
   --- Fin del seguimiento de la pila de la excepción interna ---
   en IBatisNet.DataMapper.Configuration.DomSqlMapBuilder.Build(XmlDocument document, DataSource dataSource, Boolean useConfigFileWatcher, Boolean isCallFromDao)
   en IBatisNet.DataMapper.Configuration.DomSqlMapBuilder.ConfigureAndWatch(String resource, ConfigureHandler configureDelegate)
   en PE.GOB.FSD.DataAccess.Mapper.MapperPro.InitMapper() en D:\proynet\fsd2\SBS.UIF.CONTRALAFT.DataAccess\Mapper\MapperPro.cs:línea 20
   en PE.GOB.FSD.DataAccess.Mapper.MapperPro.Instance() en D:\proynet\fsd2\SBS.UIF.CONTRALAFT.DataAccess\Mapper\MapperPro.cs:línea 31
   en PE.GOB.FSD.DataAccess.BaseService`1.QueryForObject(String nombreMetodo, Object map, String mapper) en D:\proynet\fsd2\SBS.UIF.CONTRALAFT.DataAccess\BaseDatos.cs:línea 20
   en PE.GOB.FSD.DataAccess.Core.UsuarioDataAccess.buscarUsuario(Usuario _usuario) en D:\proynet\fsd2\SBS.UIF.CONTRALAFT.DataAccess\Core\UsuarioDataAccess.cs:línea 19
   en PE.GOB.FSD.BusinessLogic.Core.UsuarioBusinessLogic.buscarUsuario(Usuario _usuario) en D:\proynet\fsd2\SBS.UIF.CONTRALAFT.BusinessLogic\Core\UsuarioBusinessLogic.cs:línea 27
   en PE.GOB.FSD.Web.pages.Login.Submit_Login(Object sender, EventArgs e) en D:\proynet\fsd2\SBS.UIF.CONTRALAFT.Web\pages\login.aspx.cs:línea 30 
- The error occurred while loading SqlMap.
- initialize type alias
- The error occurred in <sqlMap resource="Recursos/core/Usuario.xml" xmlns="http://ibatis.apache.org/dataMapper" />.  
- Check the PE.GOB.FSD.Entity.Core.Usuario,PE.GOB.FSD.Entity.
2020-08-22 09:05:02.8191 ERROR IBatisNet.Common.Exceptions.ConfigurationException: 
- The error occurred while loading SqlMap.
- initialize type alias
- The error occurred in <sqlMap resource="Recursos/core/Usuario.xml" xmlns="http://ibatis.apache.org/dataMapper" />.  
- Check the PE.GOB.FSD.Entity.Core.Usuario,PE.GOB.FSD.Entity. ---> System.TypeLoadException: Could not load type from string value 'PE.GOB.FSD.Entity.Core.Usuario,PE.GOB.FSD.Entity'. ---> System.IO.FileNotFoundException: No se puede cargar el archivo o ensamblado 'PE.GOB.FSD.Entity' ni una de sus dependencias. El sistema no puede encontrar el archivo especificado.
   en System.Reflection.RuntimeAssembly._nLoad(AssemblyName fileName, String codeBase, Evidence assemblySecurity, RuntimeAssembly locationHint, StackCrawlMark& stackMark, IntPtr pPrivHostBinder, Boolean throwOnFileNotFound, Boolean forIntrospection, Boolean suppressSecurityChecks)
   en System.Reflection.RuntimeAssembly.nLoad(AssemblyName fileName, String codeBase, Evidence assemblySecurity, RuntimeAssembly locationHint, StackCrawlMark& stackMark, IntPtr pPrivHostBinder, Boolean throwOnFileNotFound, Boolean forIntrospection, Boolean suppressSecurityChecks)
   en System.Reflection.RuntimeAssembly.InternalLoadAssemblyName(AssemblyName assemblyRef, Evidence assemblySecurity, RuntimeAssembly reqAssembly, StackCrawlMark& stackMark, IntPtr pPrivHostBinder, Boolean throwOnFileNotFound, Boolean forIntrospection, Boolean suppressSecurityChecks)
   en System.Reflection.RuntimeAssembly.InternalLoad(String assemblyString, Evidence assemblySecurity, StackCrawlMark& stackMark, IntPtr pPrivHostBinder, Boolean forIntrospection)
   en System.Reflection.RuntimeAssembly.InternalLoad(String assemblyString, Evidence assemblySecurity, StackCrawlMark& stackMark, Boolean forIntrospection)
   en System.Reflection.Assembly.Load(String assemblyString)
   en IBatisNet.Common.Utilities.TypesResolver.TypeResolver.LoadTypeDirectlyFromAssembly(TypeAssemblyInfo typeInfo)
   en IBatisNet.Common.Utilities.TypesResolver.TypeResolver.ResolveType(String typeName)
   --- Fin del seguimiento de la pila de la excepción interna ---
   en IBatisNet.Common.Utilities.TypesResolver.TypeResolver.ResolveType(String typeName)
   en IBatisNet.Common.Utilities.TypesResolver.TypeResolver.Resolve(String typeName)
   en IBatisNet.Common.Utilities.TypesResolver.CachedTypeResolver.Resolve(String typeName)
   en IBatisNet.Common.Utilities.TypeUtils.ResolveType(String typeName)
   en IBatisNet.DataMapper.Configuration.Serializers.TypeAliasDeSerializer.Deserialize(XmlNode node, ConfigurationScope configScope)
   en IBatisNet.DataMapper.Configuration.DomSqlMapBuilder.ConfigureSqlMap()
   en IBatisNet.DataMapper.Configuration.DomSqlMapBuilder.Initialize()
   en IBatisNet.DataMapper.Configuration.DomSqlMapBuilder.Build(XmlDocument document, DataSource dataSource, Boolean useConfigFileWatcher, Boolean isCallFromDao)
   --- Fin del seguimiento de la pila de la excepción interna ---
   en IBatisNet.DataMapper.Configuration.DomSqlMapBuilder.Build(XmlDocument document, DataSource dataSource, Boolean useConfigFileWatcher, Boolean isCallFromDao)
   en IBatisNet.DataMapper.Configuration.DomSqlMapBuilder.ConfigureAndWatch(String resource, ConfigureHandler configureDelegate)
   en PE.GOB.FSD.DataAccess.Mapper.MapperPro.InitMapper() en D:\proynet\fsd2\SBS.UIF.CONTRALAFT.DataAccess\Mapper\MapperPro.cs:línea 20
   en PE.GOB.FSD.DataAccess.Mapper.MapperPro.Instance() en D:\proynet\fsd2\SBS.UIF.CONTRALAFT.DataAccess\Mapper\MapperPro.cs:línea 31
   en PE.GOB.FSD.DataAccess.BaseService`1.QueryForObject(String nombreMetodo, Object map, String mapper) en D:\proynet\fsd2\SBS.UIF.CONTRALAFT.DataAccess\BaseDatos.cs:línea 20
   en PE.GOB.FSD.DataAccess.Core.UsuarioDataAccess.buscarUsuario(Usuario _usuario) en D:\proynet\fsd2\SBS.UIF.CONTRALAFT.DataAccess\Core\UsuarioDataAccess.cs:línea 19
   en PE.GOB.FSD.BusinessLogic.Core.UsuarioBusinessLogic.buscarUsuario(Usuario _usuario) en D:\proynet\fsd2\SBS.UIF.CONTRALAFT.BusinessLogic\Core\UsuarioBusinessLogic.cs:línea 27
   en PE.GOB.FSD.Web.pages.Login.Submit_Login(Object sender, EventArgs e) en D:\proynet\fsd2\SBS.UIF.CONTRALAFT.Web\pages\login.aspx.cs:línea 30 
- The error occurred while loading SqlMap.
- initialize type alias
- The error occurred in <sqlMap resource="Recursos/core/Usuario.xml" xmlns="http://ibatis.apache.org/dataMapper" />.  
- Check the PE.GOB.FSD.Entity.Core.Usuario,PE.GOB.FSD.Entity.
2020-08-22 10:50:53.1477 ERROR IBatisNet.Common.Exceptions.ConfigurationException: 
- The error occurred while loading SqlMap.
- initialize type alias
- The error occurred in <sqlMap resource="Recursos/core/Usuario.xml" xmlns="http://ibatis.apache.org/dataMapper" />.  
- Check the PE.GOB.FSD.Entity.Core.Usuario,PE.GOB.FSD.Entity. ---> System.TypeLoadException: Could not load type from string value 'PE.GOB.FSD.Entity.Core.Usuario,PE.GOB.FSD.Entity'. ---> System.IO.FileNotFoundException: No se puede cargar el archivo o ensamblado 'PE.GOB.FSD.Entity' ni una de sus dependencias. El sistema no puede encontrar el archivo especificado.
   en System.Reflection.RuntimeAssembly._nLoad(AssemblyName fileName, String codeBase, Evidence assemblySecurity, RuntimeAssembly locationHint, StackCrawlMark& stackMark, IntPtr pPrivHostBinder, Boolean throwOnFileNotFound, Boolean forIntrospection, Boolean suppressSecurityChecks)
   en System.Reflection.RuntimeAssembly.nLoad(AssemblyName fileName, String codeBase, Evidence assemblySecurity, RuntimeAssembly locationHint, StackCrawlMark& stackMark, IntPtr pPrivHostBinder, Boolean throwOnFileNotFound, Boolean forIntrospection, Boolean suppressSecurityChecks)
   en System.Reflection.RuntimeAssembly.InternalLoadAssemblyName(AssemblyName assemblyRef, Evidence assemblySecurity, RuntimeAssembly reqAssembly, StackCrawlMark& stackMark, IntPtr pPrivHostBinder, Boolean throwOnFileNotFound, Boolean forIntrospection, Boolean suppressSecurityChecks)
   en System.Reflection.RuntimeAssembly.InternalLoad(String assemblyString, Evidence assemblySecurity, StackCrawlMark& stackMark, IntPtr pPrivHostBinder, Boolean forIntrospection)
   en System.Reflection.RuntimeAssembly.InternalLoad(String assemblyString, Evidence assemblySecurity, StackCrawlMark& stackMark, Boolean forIntrospection)
   en System.Reflection.Assembly.Load(String assemblyString)
   en IBatisNet.Common.Utilities.TypesResolver.TypeResolver.LoadTypeDirectlyFromAssembly(TypeAssemblyInfo typeInfo)
   en IBatisNet.Common.Utilities.TypesResolver.TypeResolver.ResolveType(String typeName)
   --- Fin del seguimiento de la pila de la excepción interna ---
   en IBatisNet.Common.Utilities.TypesResolver.TypeResolver.ResolveType(String typeName)
   en IBatisNet.Common.Utilities.TypesResolver.TypeResolver.Resolve(String typeName)
   en IBatisNet.Common.Utilities.TypesResolver.CachedTypeResolver.Resolve(String typeName)
   en IBatisNet.Common.Utilities.TypeUtils.ResolveType(String typeName)
   en IBatisNet.DataMapper.Configuration.Serializers.TypeAliasDeSerializer.Deserialize(XmlNode node, ConfigurationScope configScope)
   en IBatisNet.DataMapper.Configuration.DomSqlMapBuilder.ConfigureSqlMap()
   en IBatisNet.DataMapper.Configuration.DomSqlMapBuilder.Initialize()
   en IBatisNet.DataMapper.Configuration.DomSqlMapBuilder.Build(XmlDocument document, DataSource dataSource, Boolean useConfigFileWatcher, Boolean isCallFromDao)
   --- Fin del seguimiento de la pila de la excepción interna ---
   en IBatisNet.DataMapper.Configuration.DomSqlMapBuilder.Build(XmlDocument document, DataSource dataSource, Boolean useConfigFileWatcher, Boolean isCallFromDao)
   en IBatisNet.DataMapper.Configuration.DomSqlMapBuilder.ConfigureAndWatch(String resource, ConfigureHandler configureDelegate)
   en PE.GOB.FSD.DataAccess.Mapper.MapperPro.InitMapper() en D:\proynet\fsd2\SBS.UIF.CONTRALAFT.DataAccess\Mapper\MapperPro.cs:línea 20
   en PE.GOB.FSD.DataAccess.Mapper.MapperPro.Instance() en D:\proynet\fsd2\SBS.UIF.CONTRALAFT.DataAccess\Mapper\MapperPro.cs:línea 31
   en PE.GOB.FSD.DataAccess.BaseService`1.QueryForObject(String nombreMetodo, Object map, String mapper) en D:\proynet\fsd2\SBS.UIF.CONTRALAFT.DataAccess\BaseDatos.cs:línea 20
   en PE.GOB.FSD.DataAccess.Core.UsuarioDataAccess.buscarUsuario(Usuario _usuario) en D:\proynet\fsd2\SBS.UIF.CONTRALAFT.DataAccess\Core\UsuarioDataAccess.cs:línea 19
   en PE.GOB.FSD.BusinessLogic.Core.UsuarioBusinessLogic.buscarUsuario(Usuario _usuario) en D:\proynet\fsd2\SBS.UIF.CONTRALAFT.BusinessLogic\Core\UsuarioBusinessLogic.cs:línea 27
   en PE.GOB.FSD.Web.pages.Login.Submit_Login(Object sender, EventArgs e) en D:\proynet\fsd2\SBS.UIF.CONTRALAFT.Web\pages\login.aspx.cs:línea 30 
- The error occurred while loading SqlMap.
- initialize type alias
- The error occurred in <sqlMap resource="Recursos/core/Usuario.xml" xmlns="http://ibatis.apache.org/dataMapper" />.  
- Check the PE.GOB.FSD.Entity.Core.Usuario,PE.GOB.FSD.Entity.
2020-08-22 10:50:53.1078 ERROR IBatisNet.Common.Exceptions.ConfigurationException: 
- The error occurred while loading SqlMap.
- initialize type alias
- The error occurred in <sqlMap resource="Recursos/core/Usuario.xml" xmlns="http://ibatis.apache.org/dataMapper" />.  
- Check the PE.GOB.FSD.Entity.Core.Usuario,PE.GOB.FSD.Entity. ---> System.TypeLoadException: Could not load type from string value 'PE.GOB.FSD.Entity.Core.Usuario,PE.GOB.FSD.Entity'. ---> System.IO.FileNotFoundException: No se puede cargar el archivo o ensamblado 'PE.GOB.FSD.Entity' ni una de sus dependencias. El sistema no puede encontrar el archivo especificado.
   en System.Reflection.RuntimeAssembly._nLoad(AssemblyName fileName, String codeBase, Evidence assemblySecurity, RuntimeAssembly locationHint, StackCrawlMark& stackMark, IntPtr pPrivHostBinder, Boolean throwOnFileNotFound, Boolean forIntrospection, Boolean suppressSecurityChecks)
   en System.Reflection.RuntimeAssembly.nLoad(AssemblyName fileName, String codeBase, Evidence assemblySecurity, RuntimeAssembly locationHint, StackCrawlMark& stackMark, IntPtr pPrivHostBinder, Boolean throwOnFileNotFound, Boolean forIntrospection, Boolean suppressSecurityChecks)
   en System.Reflection.RuntimeAssembly.InternalLoadAssemblyName(AssemblyName assemblyRef, Evidence assemblySecurity, RuntimeAssembly reqAssembly, StackCrawlMark& stackMark, IntPtr pPrivHostBinder, Boolean throwOnFileNotFound, Boolean forIntrospection, Boolean suppressSecurityChecks)
   en System.Reflection.RuntimeAssembly.InternalLoad(String assemblyString, Evidence assemblySecurity, StackCrawlMark& stackMark, IntPtr pPrivHostBinder, Boolean forIntrospection)
   en System.Reflection.RuntimeAssembly.InternalLoad(String assemblyString, Evidence assemblySecurity, StackCrawlMark& stackMark, Boolean forIntrospection)
   en System.Reflection.Assembly.Load(String assemblyString)
   en IBatisNet.Common.Utilities.TypesResolver.TypeResolver.LoadTypeDirectlyFromAssembly(TypeAssemblyInfo typeInfo)
   en IBatisNet.Common.Utilities.TypesResolver.TypeResolver.ResolveType(String typeName)
   --- Fin del seguimiento de la pila de la excepción interna ---
   en IBatisNet.Common.Utilities.TypesResolver.TypeResolver.ResolveType(String typeName)
   en IBatisNet.Common.Utilities.TypesResolver.TypeResolver.Resolve(String typeName)
   en IBatisNet.Common.Utilities.TypesResolver.CachedTypeResolver.Resolve(String typeName)
   en IBatisNet.Common.Utilities.TypeUtils.ResolveType(String typeName)
   en IBatisNet.DataMapper.Configuration.Serializers.TypeAliasDeSerializer.Deserialize(XmlNode node, ConfigurationScope configScope)
   en IBatisNet.DataMapper.Configuration.DomSqlMapBuilder.ConfigureSqlMap()
   en IBatisNet.DataMapper.Configuration.DomSqlMapBuilder.Initialize()
   en IBatisNet.DataMapper.Configuration.DomSqlMapBuilder.Build(XmlDocument document, DataSource dataSource, Boolean useConfigFileWatcher, Boolean isCallFromDao)
   --- Fin del seguimiento de la pila de la excepción interna ---
   en IBatisNet.DataMapper.Configuration.DomSqlMapBuilder.Build(XmlDocument document, DataSource dataSource, Boolean useConfigFileWatcher, Boolean isCallFromDao)
   en IBatisNet.DataMapper.Configuration.DomSqlMapBuilder.ConfigureAndWatch(String resource, ConfigureHandler configureDelegate)
   en PE.GOB.FSD.DataAccess.Mapper.MapperPro.InitMapper() en D:\proynet\fsd2\SBS.UIF.CONTRALAFT.DataAccess\Mapper\MapperPro.cs:línea 20
   en PE.GOB.FSD.DataAccess.Mapper.MapperPro.Instance() en D:\proynet\fsd2\SBS.UIF.CONTRALAFT.DataAccess\Mapper\MapperPro.cs:línea 31
   en PE.GOB.FSD.DataAccess.BaseService`1.QueryForObject(String nombreMetodo, Object map, String mapper) en D:\proynet\fsd2\SBS.UIF.CONTRALAFT.DataAccess\BaseDatos.cs:línea 20
   en PE.GOB.FSD.DataAccess.Core.UsuarioDataAccess.buscarUsuario(Usuario _usuario) en D:\proynet\fsd2\SBS.UIF.CONTRALAFT.DataAccess\Core\UsuarioDataAccess.cs:línea 19
   en PE.GOB.FSD.BusinessLogic.Core.UsuarioBusinessLogic.buscarUsuario(Usuario _usuario) en D:\proynet\fsd2\SBS.UIF.CONTRALAFT.BusinessLogic\Core\UsuarioBusinessLogic.cs:línea 27
   en PE.GOB.FSD.Web.pages.Login.Submit_Login(Object sender, EventArgs e) en D:\proynet\fsd2\SBS.UIF.CONTRALAFT.Web\pages\login.aspx.cs:línea 30 
- The error occurred while loading SqlMap.
- initialize type alias
- The error occurred in <sqlMap resource="Recursos/core/Usuario.xml" xmlns="http://ibatis.apache.org/dataMapper" />.  
- Check the PE.GOB.FSD.Entity.Core.Usuario,PE.GOB.FSD.Entity.
2020-08-22 11:05:16.0141 ERROR System.Threading.ThreadAbortException: Subproceso anulado.
   en System.Threading.Thread.AbortInternal()
   en System.Threading.Thread.Abort(Object stateInfo)
   en System.Web.HttpResponse.AbortCurrentThread()
   en System.Web.HttpResponse.End()
   en System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)
   en System.Web.HttpResponse.Redirect(String url)
   en PE.GOB.FSD.Web.pages.Login.Submit_Login(Object sender, EventArgs e) en D:\proynet\fsd2\SBS.UIF.CONTRALAFT.Web\pages\login.aspx.cs:línea 32 Subproceso anulado.
2020-08-22 23:26:08.4632 ERROR IBatisNet.DataMapper.Exceptions.DataMapperException: Unable to open connection to "Microsoft SQL Server, provider V4.0.0.0 in framework .NET V4.0". ---> System.Data.SqlClient.SqlException: Error relacionado con la red o específico de la instancia mientras se establecía una conexión con el servidor SQL Server. No se encontró el servidor o éste no estaba accesible. Compruebe que el nombre de la instancia es correcto y que SQL Server está configurado para admitir conexiones remotas. (provider: Named Pipes Provider, error: 40 - No se pudo abrir una conexión con SQL Server) ---> System.ComponentModel.Win32Exception: El sistema no puede encontrar el archivo especificado
   --- Fin del seguimiento de la pila de la excepción interna ---
   en System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, DbConnectionPool pool, String accessToken, Boolean applyTransientFaultHandling, SqlAuthenticationProviderManager sqlAuthProviderManager)
   en System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   en System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   en System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   en System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   en System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   en System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   en System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   en System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   en System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   en System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   en System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   en System.Data.SqlClient.SqlConnection.Open()
   en IBatisNet.DataMapper.SqlMapSession.OpenConnection(String connectionString)
   --- Fin del seguimiento de la pila de la excepción interna ---
   en IBatisNet.DataMapper.SqlMapSession.OpenConnection(String connectionString)
   en IBatisNet.DataMapper.SqlMapSession.OpenConnection()
   en IBatisNet.DataMapper.Commands.DbCommandDecorator.System.Data.IDbCommand.ExecuteReader()
   en IBatisNet.DataMapper.MappedStatements.MappedStatement.RunQueryForObject(RequestScope request, ISqlMapSession session, Object parameterObject, Object resultObject)
   en IBatisNet.DataMapper.MappedStatements.MappedStatement.ExecuteQueryForObject(ISqlMapSession session, Object parameterObject, Object resultObject)
   en IBatisNet.DataMapper.MappedStatements.MappedStatement.ExecuteQueryForObject(ISqlMapSession session, Object parameterObject)
   en IBatisNet.DataMapper.SqlMapper.QueryForObject(String statementName, Object parameterObject)
   en PE.GOB.FSD.DataAccess.BaseService`1.QueryForObject(String nombreMetodo, Object map, String mapper) en D:\proynet\fsd3\SBS.UIF.CONTRALAFT.DataAccess\BaseDatos.cs:línea 20
   en PE.GOB.FSD.DataAccess.Core.UsuarioDataAccess.buscarUsuario(Usuario _usuario) en D:\proynet\fsd3\SBS.UIF.CONTRALAFT.DataAccess\Core\UsuarioDataAccess.cs:línea 19
   en PE.GOB.FSD.BusinessLogic.Core.UsuarioBusinessLogic.buscarUsuario(Usuario _usuario) en D:\proynet\fsd3\SBS.UIF.CONTRALAFT.BusinessLogic\Core\UsuarioBusinessLogic.cs:línea 27
   en PE.GOB.FSD.Web.pages.Login.Submit_Login(Object sender, EventArgs e) en D:\proynet\fsd3\SBS.UIF.CONTRALAFT.Web\pages\login.aspx.cs:línea 30 Unable to open connection to "Microsoft SQL Server, provider V4.0.0.0 in framework .NET V4.0".
2020-08-22 23:29:34.6543 ERROR IBatisNet.DataMapper.Exceptions.DataMapperException: Unable to open connection to "Microsoft SQL Server, provider V4.0.0.0 in framework .NET V4.0". ---> System.Data.SqlClient.SqlException: Error relacionado con la red o específico de la instancia mientras se establecía una conexión con el servidor SQL Server. No se encontró el servidor o éste no estaba accesible. Compruebe que el nombre de la instancia es correcto y que SQL Server está configurado para admitir conexiones remotas. (provider: Named Pipes Provider, error: 40 - No se pudo abrir una conexión con SQL Server) ---> System.ComponentModel.Win32Exception: El sistema no puede encontrar el archivo especificado
   --- Fin del seguimiento de la pila de la excepción interna ---
   en System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, DbConnectionPool pool, String accessToken, Boolean applyTransientFaultHandling, SqlAuthenticationProviderManager sqlAuthProviderManager)
   en System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   en System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   en System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   en System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   en System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   en System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   en System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   en System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   en System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   en System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   en System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   en System.Data.SqlClient.SqlConnection.Open()
   en IBatisNet.DataMapper.SqlMapSession.OpenConnection(String connectionString)
   --- Fin del seguimiento de la pila de la excepción interna ---
   en IBatisNet.DataMapper.SqlMapSession.OpenConnection(String connectionString)
   en IBatisNet.DataMapper.SqlMapSession.OpenConnection()
   en IBatisNet.DataMapper.Commands.DbCommandDecorator.System.Data.IDbCommand.ExecuteReader()
   en IBatisNet.DataMapper.MappedStatements.MappedStatement.RunQueryForObject(RequestScope request, ISqlMapSession session, Object parameterObject, Object resultObject)
   en IBatisNet.DataMapper.MappedStatements.MappedStatement.ExecuteQueryForObject(ISqlMapSession session, Object parameterObject, Object resultObject)
   en IBatisNet.DataMapper.MappedStatements.MappedStatement.ExecuteQueryForObject(ISqlMapSession session, Object parameterObject)
   en IBatisNet.DataMapper.SqlMapper.QueryForObject(String statementName, Object parameterObject)
   en PE.GOB.FSD.DataAccess.BaseService`1.QueryForObject(String nombreMetodo, Object map, String mapper) en D:\proynet\fsd3\SBS.UIF.CONTRALAFT.DataAccess\BaseDatos.cs:línea 20
   en PE.GOB.FSD.DataAccess.Core.UsuarioDataAccess.buscarUsuario(Usuario _usuario) en D:\proynet\fsd3\SBS.UIF.CONTRALAFT.DataAccess\Core\UsuarioDataAccess.cs:línea 19
   en PE.GOB.FSD.BusinessLogic.Core.UsuarioBusinessLogic.buscarUsuario(Usuario _usuario) en D:\proynet\fsd3\SBS.UIF.CONTRALAFT.BusinessLogic\Core\UsuarioBusinessLogic.cs:línea 27
   en PE.GOB.FSD.Web.pages.Login.Submit_Login(Object sender, EventArgs e) en D:\proynet\fsd3\SBS.UIF.CONTRALAFT.Web\pages\login.aspx.cs:línea 30 Unable to open connection to "Microsoft SQL Server, provider V4.0.0.0 in framework .NET V4.0".
2020-08-22 23:31:26.1939 ERROR IBatisNet.DataMapper.Exceptions.DataMapperException: Unable to open connection to "Microsoft SQL Server, provider V4.0.0.0 in framework .NET V4.0". ---> System.Data.SqlClient.SqlException: Error relacionado con la red o específico de la instancia mientras se establecía una conexión con el servidor SQL Server. No se encontró el servidor o éste no estaba accesible. Compruebe que el nombre de la instancia es correcto y que SQL Server está configurado para admitir conexiones remotas. (provider: Named Pipes Provider, error: 40 - No se pudo abrir una conexión con SQL Server) ---> System.ComponentModel.Win32Exception: El sistema no puede encontrar el archivo especificado
   --- Fin del seguimiento de la pila de la excepción interna ---
   en System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, DbConnectionPool pool, String accessToken, Boolean applyTransientFaultHandling, SqlAuthenticationProviderManager sqlAuthProviderManager)
   en System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   en System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   en System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   en System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   en System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   en System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   en System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   en System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   en System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   en System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   en System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   en System.Data.SqlClient.SqlConnection.Open()
   en IBatisNet.DataMapper.SqlMapSession.OpenConnection(String connectionString)
   --- Fin del seguimiento de la pila de la excepción interna ---
   en IBatisNet.DataMapper.SqlMapSession.OpenConnection(String connectionString)
   en IBatisNet.DataMapper.SqlMapSession.OpenConnection()
   en IBatisNet.DataMapper.Commands.DbCommandDecorator.System.Data.IDbCommand.ExecuteReader()
   en IBatisNet.DataMapper.MappedStatements.MappedStatement.RunQueryForObject(RequestScope request, ISqlMapSession session, Object parameterObject, Object resultObject)
   en IBatisNet.DataMapper.MappedStatements.MappedStatement.ExecuteQueryForObject(ISqlMapSession session, Object parameterObject, Object resultObject)
   en IBatisNet.DataMapper.MappedStatements.MappedStatement.ExecuteQueryForObject(ISqlMapSession session, Object parameterObject)
   en IBatisNet.DataMapper.SqlMapper.QueryForObject(String statementName, Object parameterObject)
   en PE.GOB.FSD.DataAccess.BaseService`1.QueryForObject(String nombreMetodo, Object map, String mapper) en D:\proynet\fsd3\SBS.UIF.CONTRALAFT.DataAccess\BaseDatos.cs:línea 20
   en PE.GOB.FSD.DataAccess.Core.UsuarioDataAccess.buscarUsuario(Usuario _usuario) en D:\proynet\fsd3\SBS.UIF.CONTRALAFT.DataAccess\Core\UsuarioDataAccess.cs:línea 19
   en PE.GOB.FSD.BusinessLogic.Core.UsuarioBusinessLogic.buscarUsuario(Usuario _usuario) en D:\proynet\fsd3\SBS.UIF.CONTRALAFT.BusinessLogic\Core\UsuarioBusinessLogic.cs:línea 27
   en PE.GOB.FSD.Web.pages.Login.Submit_Login(Object sender, EventArgs e) en D:\proynet\fsd3\SBS.UIF.CONTRALAFT.Web\pages\login.aspx.cs:línea 30 Unable to open connection to "Microsoft SQL Server, provider V4.0.0.0 in framework .NET V4.0".
2020-08-22 23:35:59.9226 ERROR System.Threading.ThreadAbortException: Subproceso anulado.
   en System.Threading.Thread.AbortInternal()
   en System.Threading.Thread.Abort(Object stateInfo)
   en System.Web.HttpResponse.AbortCurrentThread()
   en System.Web.HttpResponse.End()
   en System.Web.HttpResponse.Redirect(String url, Boolean endResponse, Boolean permanent)
   en System.Web.HttpResponse.Redirect(String url)
   en PE.GOB.FSD.Web.pages.Login.Submit_Login(Object sender, EventArgs e) en D:\proynet\fsd3\SBS.UIF.CONTRALAFT.Web\pages\login.aspx.cs:línea 32 Subproceso anulado.
